compiler-rt/test/fuzzer/afl-driver-stderr.test

```diff
@@ -7,7 +7,8 @@ RUN: %no_fuzzer_cpp_compiler %S/AFLDriverTest.cpp %libfuzzer_src/afl/afl_driver.
 RUN: env -u AFL_DRIVER_STDERR_DUPLICATE_FILENAME %run %t-AFLDriverTest
 
 ; Test that specifying an invalid file causes a crash.
-RUN: env ASAN_OPTIONS= AFL_DRIVER_STDERR_DUPLICATE_FILENAME="%T" not --crash %run %t-AFLDriverTest
+RUN: mkdir -p %t.dir
```

Comment:
Why do all the other tests `rm -rf` the dir but not this?

Reply:
The other ones were deleting test artifacts originally whereas this one was not. I tried to preserve the existing behavior.